libbpfgo.go

```diff
@@ -3,77 +3,80 @@ package libbpfgo
 /*
 #cgo LDFLAGS: -lelf -lz
 
-#include <bpf/bpf.h>
-#include <bpf/libbpf.h>
-#include <stdint.h>
-#include <stdio.h>
+#include <errno.h>
+#include <fcntl.h>
 #include <stdlib.h>
 #include <sys/resource.h>
 
-#include <asm-generic/unistd.h>
-#include <errno.h>
-#include <fcntl.h>
-#include <linux/perf_event.h>
-#include <linux/unistd.h>
-#include <string.h>
-#include <unistd.h>
+#include <bpf/bpf.h>
+#include <bpf/libbpf.h>
```

Comment:
Are these changes related to this commit or the other one(removing the kprobe)?
We better put it in the relevant commit for the sake of git history

Reply:
No need to create a new one. I just think it is more relevant for the other commit that removed the kprobes code, isn't it?